vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/AbsMessageItem.kt

```diff
@@ -94,6 +94,11 @@ abstract class AbsMessageItem<H : AbsMessageItem.Holder> : AbsBaseMessageItem<H>
             holder.memberNameView.setTextColor(attributes.getMemberNameColor())
             holder.memberNameView.onClick(_memberNameClickListener)
             holder.memberNameView.setOnLongClickListener(attributes.itemLongClickListener)
+        } else if (attributes.informationData.messageLayout.marge) {
+            holder.memberNameView.setOnClickListener(null)
+            holder.memberNameView.setOnLongClickListener(null)
+            holder.memberNameView.isVisible = true
+            holder.memberNameView.text = ""
```

Comment:
Maybe using `isInvisible` like `holder.memberNameView.isInvisible = true` is enough, but this should also work